server/build.gradle

```diff
@@ -60,7 +60,6 @@ dependencies {
     implementation('com.samskivert:jmustache:1.16')
     implementation('commons-chain:commons-chain:1.2')
     implementation('org.apache.commons:commons-fileupload2-jakarta:2.0.0-M1')
-    implementation('commons-io:commons-io:2.12.0')
```

Comment:
Explicitly pin version leads to mismatch in transitive dependencies. `org.apache.commons:commons-compress` requires at least commons-io:commons-io:2.16.x